Library/ViewModels/PledgeViewModel.swift

```diff
@@ -20,6 +20,7 @@ public typealias PledgeViewData = (
 public protocol PledgeViewModelInputs {
   func configureWith(project: Project, reward: Reward)
   func pledgeAmountDidUpdate(to amount: Double)
+  func presentShippingRules(with rule: ShippingRule)
```

Comment:
Typically we want our inputs to represent _what happened_, and our outputs to be named according to _what should happen_, so in this case our output is `presentShippingRules` because that's the side effect, but our input should be more like exactly what happened - `pledgeShippingCellWillPresentShippingRules` which is matches our delegate naming.